brainstorm/src/components/items.tsx

```diff
@@ -78,15 +92,27 @@ export function ItemsView(props: {
 	}
 }
 
-/**
- * Removes a node from its parent {@link Items}.
- * If the note is not in an {@link Items}, it is left unchanged.
- */
-export function deleteItem(item: Item): void {
-	const parent = Tree.parent(item);
-	// Use type narrowing to ensure that parent is Items as expected for an Item.
-	if (Tree.is(parent, Items)) {
-		const index = parent.indexOf(item);
-		parent.removeAt(index);
+export interface ItemParent {
+	/**
+	 * Removes a child Item.
+	 */
+	deleteItem(item: Item): void;
+}
+
+interface HasItemParent extends TreeNode {
+	get [ItemParentSymbol](): ItemParent;
```

Comment:
in the interface, you might as well just do 
```suggestion
	readonly [ItemParentSymbol]: ItemParent;
```

Reply:
Done.